spec/features/admin_pages_spec.rb

```diff
@@ -69,37 +69,34 @@
   end
 end
 
-feature 'Inviting new users' do
+feature 'Inviting new users', :vcr do
   let(:admin) { FactoryGirl.create(:admin) }
-  let(:course) { FactoryGirl.build(:course) }
+  let!(:course) { FactoryGirl.create(:course, description: '* Placement Test') }
```

Comment:
Apparently let without the ! does not actually instantiate the object unless the object is explicitly called in the test, and these tests rely on the course's existence but the test does not say something like expect(course).